### PR TITLE
Use uglify-es 3.2.2 to solve critical transitive dependency bug

### PR DESCRIPTION
**Pull request by pascalgrimaud**
It should fix https://github.com/jhipster/generator-jhipster/issues/6910

Tested with Yarn and NPM : see my comments in the code for details

_____

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/_package.json [hunk 1]
```diff
@@ -121,6 +121,7 @@
     "<%= module.name %>": "<%= module.version %>",
     <%_ }); _%>
     "uglifyjs-webpack-plugin": "1.1.4",
+    "uglify-es": "3.2.1",
```
Comment (pascalgrimaud) [inline generators/client/templates/angular/_package.json]: **Yarn** doesn't care about that (see Deepu's comment: https://github.com/jhipster/generator-jhipster/issues/6910#issuecomment-353992198)

This is for NPM only. With that:
```
09:28:07 in ~/tmp/29-npm on  master 
➜ npm ls uglify-es
jhipster@0.0.0 /home/pgrimaud/tmp/29-npm
├── uglify-es@3.2.1 
└─┬ uglifyjs-webpack-plugin@1.1.4
  └── uglify-es@3.2.1  deduped
```

File: generators/client/templates/angular/_package.json [hunk 2]
```diff
@@ -130,6 +131,9 @@
     "workbox-webpack-plugin": "2.1.2",
     "write-file-webpack-plugin": "4.1.0"
   },
+  "resolutions": {
```
Comment (pascalgrimaud) [inline generators/client/templates/angular/_package.json]: This part is for **Yarn**

**Comment by jdubois** [comment]
Version 3.2.2 should also work

**Comment by pascalgrimaud** [comment]
Ok let me upgrade to 3.2.2 and see if Travis is happy

**Comment by jdubois** [comment]
It's green, let's go!!!!

**Comment by jdubois** [comment]
@pascalgrimaud now that uglifyjs-webpack-plugin merged my PR https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/199 I'm going to revert your commit, and point to uglifyjs-webpack-plugin v1.1.5 instead.
That's what Angular CLI is doing https://github.com/angular/angular-cli/pull/9006/files
(so basically JHipster corrected Angular CLI!!)

**Comment by pascalgrimaud** [comment]
good to me @jdubois !

**Pull request merged**